tests/helpers/network.ts

```diff
@@ -68,8 +68,11 @@ export const visitNetworkConfiguration = async (page: Page, tab: string) => {
   await activateAllTableOverrides(page);
 };
 
-export const createNetworkForward = async (page: Page, network: string) => {
-  await createNetwork(page, network);
+export const createNetworkForward = async (
+  page: Page,
+  network: string,
+  serverClustered?: boolean,
```

Comment:
```suggestion
```
I think it will be simpler to make the call to `isServerClustered` inside this function instead of expanding the interface here and expect the caller to pass this in.